lazy-load.js

```diff
@@ -76,7 +76,7 @@ function disconnect() {
 function onIntersection(entries) {
   // Disconnect if we've already loaded all of the images
   if (imageCount === 0) {
-    observer.disconnect();
+    disconnect();
```

Comment:
But I do, since I am calling the disconnect function (that uses observer.disconnect) which is right above this one... Or do I miss something?